controller/src/beerocks/cli/beerocks_cli_bml.cpp

```diff
@@ -1307,6 +1321,49 @@ int cli_bml::get_dcs_scan_results_caller(int numOfArgs)
     return -1;
 }
 
+/**
+ * Caller function for client_get_client_list_caller.
+ *
+ * @param [in] numOfArgs Number of received arguments
+ * @return 0 on success.
+ */
+int cli_bml::client_get_client_list_caller(int numOfArgs)
+{
+    if (numOfArgs == 0) {
```

Comment:
we might need to think of an interface that provides "max-number-of-client-to-get" or "returned-clients-buffer-max-size" similar to the get-scan-results in the DCS.
@kantera800 what do you think?

Reply:
TBH, the only reason we need the result count in the DCS is because of the outgoing API we aligned the BML with the client demand. a "max-number-of-client-to-get" is not needed because we would either get the client list as a vector of strings (or any other container that fits) or we concatenate the strings into a single one (like in the current code).
both cases there is no need for a size variable, since there shouldn't be a scenario where we want to get N listings when there is M available (and N < M), so we should always try and get the most listings possible (no artificial max value)

Reply:
we can add it later on if needed